tests/test_inner.py

```diff
@@ -117,7 +129,10 @@ def get_built_image_info(self):
         return {'Id': 'some'}
 
     def inspect_built_image(self):
-        return None
+        return {INSPECT_ROOTFS: {INSPECT_ROOTFS_LAYERS: ['sha256:1234',
```

Comment:
`def history` returns size for `Id: "sha256:1"` and `ROOTFS_LAYERS` returns `'sha256:1234'` - and tests pass? This is weird, must be a bug somewhere

Reply:
what?? we need diff_ids from rootfs, and sizes from history, those are different IDs, one is diff_ids and another is regular id

Reply:
History describes the layers in terms of their image IDs; inspect describes them in terms of diff_ids. Both look like "sha256:...".

Perhaps the test would look less confusing if the "sha256:" strings described themselves better, e.g.

- sha256:diff_id1
- sha256:diff_id2
- sha256:diff_id3
- ...

and

- sha256:layer0-oldest
- sha256:layer1
- sha256:layer2
- sha256:layer3-newest

Reply:
>"sha256:" strings described themselves better

good idea